packages/patterns/src/keys/checkKey.js

```diff
@@ -551,6 +551,9 @@ const checkKeyInternal = (val, check) => {
       // A copyArray is a key iff all its children are keys
       return val.every(checkIt);
     }
+    case 'byteArray': {
+      return true;
+    }
```

Comment:
I noticed you did not suggest deleting this one, which is correct. My investigating why led me to realize a bug we would have had once passable symbols are represented as js objects. `isObject` is no longer a good classifiers for whether something is a passable-primitive value. I added a TODO to the code.

Resolving.